lib/app/console_app.rb

```diff
@@ -39,7 +39,7 @@ module ConsoleApp
     def start_sync_now
       return unless connector_registered?
       puts 'Initiating synchronization NOW...'
-      connector_id = App::Config[:connector_package_id]
+      connector_id = App::Config[:connector_id]
```

Comment:
fly-by review, the connector_id field is used everywhere, maybe we can add it as a class attribute when initializing, with

```
@connector_id = App::Config[:connector_id]
```

Reply:
Good one, done in https://github.com/elastic/connectors-ruby/pull/194/commits/718867806e78c758c8d95ee4ae56106e8e23d4e4